apps/desktop-wallet/src/modals/SettingsModal/GeneralSettingsSection.tsx

```diff
@@ -127,6 +128,7 @@ const GeneralSettingsSection = ({ className }: GeneralSettingsSectionProps) => {
   const handleClearCacheButtonPress = async () => {
     try {
       queryClient.clear()
+      deleteThumbnailsDB()
```

Comment:
are we sure we want to bundle this together with the tanstack query cache? We did not add the walletconnect cache clearing together with the tanstack because one might fail whereas the other might not.